internal/oonitemplates/oonitemplates.go

```diff
@@ -173,6 +173,10 @@ func configureDNS(seed int64, network, address string) (modelx.DNSResolver, erro
 		return nil, err
 	}
 	fallbacks := []dnsFallback{
+		dnsFallback{
+			network: "doh",
+			address: "https://doh.powerdns.org/",
+		},
```

Comment:
Can you remove this part of the diff? The oonitemplates.go code is code that I am removing and it does not matter adding this bit of code here. Now only one test out of many is still using this code.

Reply:
done! :D